crates/egui_kittest/src/lib.rs

```diff
@@ -28,15 +28,20 @@ use kittest::{Node, Queryable};
 
 /// The test Harness. This contains everything needed to run the test.
 /// Create a new Harness using [`Harness::new`] or [`Harness::builder`].
-pub struct Harness<'a> {
+///
+/// The [Harness] has a optional generic state that can be used to pass data to the app / ui closure.
+/// In _most cases_ it should be fine to just store the state in the closure itself.
+/// The state functions are useful if you need to access the state after the harness has been created.
+pub struct Harness<'a, S = ()> {
```

Comment:
explicit is better imho; here and everywhere

```suggestion
pub struct Harness<'a, State= ()> {
```